docs/source/models/causes/neonatal/index.rst

```diff
@@ -257,10 +271,42 @@ Data Tables
       - Definition
       - Value or source
       - Note
-    * - :math:`\text{ACMR}`
-      - all-cause mortality rate (per person year)
-      - GBD
-      -
+    * - enn_all_cause_death_count
+      - Death count in the early neonatal age group
+      - GBD: source='codcorrect', metric_id=1, cause_id=294
+      - 
+    * - lnn_all_cause_death_count
+      - Death count in the early neonatal age group due to all causes
+      - GBD: source='codcorrect', metric_id=1, cause_id=294
+      - 
+    * - enn_cause_specific_death_count
+      - Count of deaths due to cause C in the early neonatal age group
+      - GBD: source='codcorrect', metric_id=1
+      - 
+    * - lnn_cause_specific_death_count
+      - Count of deaths due to cause C in the late neonatal age group
+      - GBD: source='codcorrect', metric_id=1
+      - 
+    * - live_birth_count
+      - Count of live births
+      - GBD: covariate_id = 1106
+      - 
+    * - enn_amr
+      - all-cause mortality risk in the early neonatal age group
+      - enn_all_cause_death_count / live_birth_count
+      - 
+    * - lnn_amr
+      - all-cause mortality risk in the late neonatal age group
+      - lnn_all_cause_death_count / (live_birth_count - enn_all_cause_death_count)
+      - 
```

Comment:
Seems like it could be good to add a row for `AMR`, defined as _either_ `enn_amr` or `lnn_amr` (or the other notation if we go with that), since `AMR` is what we actually use in our equations.